tests/e2e/scripts/cleanup.sh

```diff
@@ -40,5 +40,28 @@ kill_edgecontroller() {
     fi
 }
 
+cleanup_files(){
+    echo $PWD
+    cd $PWD
+    cd ../../
+
+    sudo rm -rf cloud/edgecontroller/edgecontroller
+    sudo rm -rf cloud/edgecontroller/edgecontroller.log
+    sudo rm -rf cloud/edgecontroller/tmp/
+    sudo rm -rf edge/edge.db
+    sudo rm -rf edge/edge_core
+    sudo rm -rf edge/edge_core.log
```

Comment:
@edisonxiang , do not cleanup module logs, as we may need them for issue debug in case if any failures.